WearOAuth/oauth-device-grant/src/main/java/com/example/android/wearable/oauth/devicegrant/AuthDeviceGrantActivity.kt

```diff
@@ -16,10 +16,27 @@
 package com.example.android.wearable.oauth.devicegrant
 
 import android.os.Bundle
-import android.view.View
-import android.widget.TextView
 import androidx.activity.ComponentActivity
-import androidx.activity.viewModels
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.wear.compose.material3.Button
```

Comment:
material3 isn't complete yet, should we stick to material?

Reply:
My thinking with using Material 3 is that I was able to implement it all without mixing Material 2.  Also we won't have to update it again once it releases.